pkg/kubelet/rkt/rkt.go

```diff
@@ -62,9 +62,9 @@ const (
 	RktType                      = "rkt"
 	DefaultRktAPIServiceEndpoint = "localhost:15441"
 
-	minimumAppcVersion       = "0.7.4"
-	minimumRktBinVersion     = "1.2.1"
-	recommendedRktBinVersion = "1.2.1"
+	minimumAppcVersion       = "0.8.1"
```

Comment:
I don't think we should need appc version since it should be strongly tied to rkt version, right?